### PR TITLE
MP5 re-work

### PR DESCRIPTION
**Pull request by tenmillimaster**
#### Summary
```SUMMARY: Content "Updates briefcase SMG, adds mp5k"```

#### Purpose of change
A throwaway million dollar briefcase makes little sense. Few materials are x-ray transparent. There is a good real-life candidate for a concealed briefcase SMG.
#### Describe the solution
Adds an H&K MP5K-PDW that fits inside the briefcase. Updates the stats on the briefcase to be very poor.

Start as a thief, cry a little, open your briefcase and wield your gun and wear your briefcase.
#### Describe alternatives you've considered
Deleting the briefcase.
#### Additional context
The foregrip and the folding stock are baked into the MP5K due to recipe difficulties.

### CODE REVIEW

**Comment by ymber** [comment]
What's the problem with using integral mods on the MP5K?

**Comment by tenmillimaster** [comment]
Nothing, but it interferes with the recipe to re-assemble the briefcase gun.

**Comment by tenmillimaster** [comment]
I see other improvements that can be made, so on WIP for a bit.

Rewriting the description for the MP5SD, changing some of the mp5 spawns, and adding semi variants.

**Comment by cjuicy12** [comment]
IRL there were briefcases that were built for Mac-10/Mac-11 concealment. There's even a part of the video of the attempt on President Reagan where one of his guards goes from holding a briefcase to a fully loaded Mac-11 in seconds.

**Comment by tenmillimaster** [comment]
> IRL there were briefcases that were built for Mac-10/Mac-11 concealment. There's even a part of the video of the attempt on President Reagan where one of his guards goes from holding a briefcase to a fully loaded Mac-11 in seconds.

Yes, but I'm not adding them.

File: data/json/items/gun/9mm.json [hunk 1]
```diff
@@ -190,6 +165,64 @@
...
+    "flags": [ "NEEDS_UNFOLD", "COLLAPSIBLE_STOCK" ]
+  },
+  {
+    "id": "hk_mp5k",
+    "type": "GUN",
+    "copy-from": "hk_mp5",
+    "name": "PTR 603",
+    "description": "The PTR 603 is a semi-automatic copy of the famous MP5 series, domestically produced for the US market on original Heckler and Koch tooling.  Like the MP5, its roller delayed blowback offers smooth shooting and precision, setting it apart from other submachine guns and pistol caliber carbines.  This model is configured as a pistol to avoid inconvenient and costly NFA regulations.  The welded on section of M1913 rail offers convenient mounting for aftermarket optics, but precludes the use of claw mounts.",
+    "relative": {
+    "weight": -300
```
Comment (LaVeyanFiend) [inline data/json/items/gun/9mm.json]:
```suggestion
    "weight": -300,
```

File: data/json/items/gun/9mm.json [hunk 2]
```diff
@@ -190,6 +165,64 @@
...
+      [ "grip", 1 ],
+      [ "mechanism", 4 ],
+      [ "rail", 1 ],
+      [ "sights", 1 ],
+      [ "sling", 1 ]
+    ],
+    "flags": [ "NEEDS_UNFOLD", "COLLAPSIBLE_STOCK" ]
+  },
+  {
+    "id": "hk_mp5k",
```
Comment (ymber) [inline data/json/items/gun/9mm.json]: ID collision with the MP5K-PDW.
```suggestion
    "id": "ptr603",
```

**Comment by ymber** [comment]
Testing with a modified version of this branch I have no issues with disassembling and recrafting the `briefcase_smg` using an `hk_mp5k` with integral grip and stock mods.  What problem did you have with it?

File: data/json/items/gun/9mm.json (hunk 2 above)
Comment (tenmillimaster) [reply data/json/items/gun/9mm.json]: Yeah, sorry, I wanted to push this so I could work on it on another machine. I actually had work to do last night so didn't get around to it

**Comment by tenmillimaster** [comment]
@ymber it's probable that I'd tested it while I had the folding stock as a default mod, not an integral one.

**Comment by tenmillimaster** [comment]
Many thanks to @ymber for his testing with integral mods, the recipe works!

I've found some data on just how loud the MP5SD is (About as loud as a suppressed glock 19), so I've nerfed it as well.

I've added the mp5's to a couple of item groups, swat and such.

File: data/json/itemgroups/guns.json [hunk 3]
```diff
@@ -479,6 +483,10 @@
     "items": [
       { "group": "guns_cop", "prob": 100 },
       { "item": "hk_ump45", "prob": 15, "charges-min": 0, "charges-max": 25 },
+      { "item": "hk_mp5", "prob": 50, "charges-min": 0, "charges-max": 30 },
+      { "item": "hk_mp5k", "prob": 10, "charges-min": 0, "charges-max": 30 },
+      { "item": "hk_mp5sd", "prob": 5, "charges-min": 0, "charges-max": 30 },
+      { "item": "hk_ump45", "prob": 30, "charges-min": 0, "charges-max": 25 },
```
Comment (ymber) [inline data/json/itemgroups/guns.json]: There's already a UMP45 in this group above.

**Pull request merged**